src/Console/Command/FixCommand.php

```diff
@@ -368,31 +373,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->setTime($fixEvent->getDuration())
         ;
 
-        $reporter = ReporterFactory::create()
-            ->registerBuiltInReporters()
-            ->getReporter($resolver->getFormat())
-        ;
-
         $output->write(
             $reporter->generate($reportSummary)
         );
 
+        if (null !== $stdErr) {
+            $stdErr = $output;
```

Comment:
ping @keradus 
The report builders should write the errors in the appropriate format. 
This is because if there is not stderr available we either dump the errors in the wrong format to the stream or do not show the errors at all, both options are not ideal.